src/d3d12/d3d12_renderer.cpp

```diff
@@ -150,6 +150,8 @@ namespace wr
 		// Execute
 		d3d12::End(m_direct_cmd_list);
 		d3d12::Execute(m_direct_queue, { m_direct_cmd_list }, m_fences[frame_idx]);
+
+		m_buffer_frame_graph_uids.resize(d3d12::settings::num_back_buffers);
```

Comment:
Does this need to be a array? Can't it just be a single value?

Reply:
I'm using an array because each of the back buffers needs to be cleared, and you can't clear them all at the same time. This allows me to know if I still need to clear this one or not.